.github/workflows/node.js.yml

```diff
@@ -1,87 +1,105 @@
-name: CI
+name: Node
 
 on:
   pull_request:
   push:
-    branches-ignore:
-      - main
-      - "dependabot/**"
+    branches-ignore: "dependabot/**"
 
 permissions:
   contents: read # to fetch code (actions/checkout)
 
 jobs:
-  build:
+  build-and-test:
```

Comment:
What about the Git builds? They are not handled in this PR yet, are they? As long as they're not, our Jenkins builds are still useful. But we should implement it in GA as well to completely get rid of the Jenkins dependency.

In the meantime, we still need Node 10 builds to work (tests not so much but at least the build).

Reply:
That is next on my list, but you're right. I'll add back Node 10 for now.